includes/languages/english/modules/payment/sagepay_zc_form.php

```diff
@@ -48,3 +48,5 @@
 
 define('TEXT_TITLE_MCRYPT_ERROR', ' (mcrypt problem)');
 define('TEXT_DESCRIPTION_MCRYPT_ERROR', ' The PHP mcrypt extension is not available. Sagepay Form will be disabled at checkout.');
+define('TEXT_TITLE_SUHOSIN_ERROR', ' (suhosin problem)');
+define('TEXT_DESCRIPTION_SUHOSIN_GET_LENGTH_ERROR', ' The Suhosin setting for the length of GET paramaeters is set to low. Please see <a target="_blank" href="https://www.zen-cart.com/content.php?342-The-Sagepay-Form-module-has-an-error-regarding-Suhosin for more details">this faq entry</a>.');
```

Comment:
`paramaeters is set to low` has 2 typos, and should be:
`parameters is set too low`